tests/Auth/AuthPasswordBrokerTest.php

```diff
@@ -115,3 +124,16 @@ protected function getMocks()
         ];
     }
 }
+
+// Before 7.x we have to check the existence of a new method. In 7.x, this code must be removed.
+
+namespace Illuminate\Auth\Passwords;
```

Comment:
I think this namespace and method shouldn't be here.

Reply:
There we need to override method_exists just for this test. Any ideas?

Reply:
I see, you should create an interface or class for this, which extends the TokenRepositoryInterface (for example TestTokenRepository) define the method on it and mock that.

```php
    public function testIfTokenIsRecentlyCreated()
    {
        // ...
        $tokenMock = m::mock(TestTokenRepository::class);
        // ...
    }
// ...
}

interface TestTokenRepository extends TokenRepositoryInterface {
    public function recentlyCreated($user);
}
```

For future reference, never do this because this can override that method if this file is loaded because of some reason (but I'm not 100% sure in this).
Furthermore you are testing the current class so you need to mock the underlying object, by "mocking" this method you basically skip part of your code that you want to test.